extensions/amp-list/amp-list.md

```diff
@@ -282,33 +282,62 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 If `binding` attribute is not provided, default is `always`.
 
 ## Experimental: Infinite Scroll (amp-list-load-more)
-We've introduced an experiment called `amp-list-load-more` as an implementation for pagination and infinite scroll in amp-list. This is an experimental feature, and final APIs may change.
+We've introduced the `amp-list-load-more` experiment as an implementation for pagination and infinite scroll in `amp-list`. This is an experimental feature, and final APIs may change.
```

Comment:
Can we include a code sample for how this is added? Is it another component inside of amp-list?

```
<amp-list>
     <amp-list-load-more>
     </amp-list-load-more>
</amp-list-load-more>
```

or added as an attribute? 

```
<amp-list amp-list-load-more>
</amp-list>
```

Reply:
Ah I see. It's because this is experimental, so you enable it on https://cdn.ampproject.org/experiments.html. I'll try to link to the relevant experiment documentation.